extension/sumologicextension/extension.go

```diff
@@ -739,6 +740,15 @@ func filteredProcessList() ([]string, error) {
 	for _, v := range processes {
 		e, err := v.Name()
 		if err != nil {
+			if runtime.GOOS == "windows" {
```

Comment:
Not sure if this would be the correct place to skip windows processes. This section of the code is to filter processes for auto discovery products (supports mac + linux).

Reply:
I'm sure if I am understanding what you meant. What this is doing is handling the case of failure to get the process name on Windows by not aborting the function and continuing with the remaing processes. 

Why is that reasonable on Windows? Because zombie processes are a relatively common ocurrence on Windows: any 3rd party process can create one by opening a handle to a process and not closing it when the other process terminates.

Why the check for Windows? Because I want to change as little as possible on the other platforms that as far as I know are not having the same issue. In principle, I don't see a problem doing the same for the other OSes the intent was to change as little as possible the current behavior. That said removing the if and just logging the error and continuing is also fine.

Reply:
@chan-tim-sumo, this PR is intended to stabilize and fix CI pipeline on Windows. If you think it's not the right fix, please suggest your solution as another PR. In the meantime, since there is no agreement with the codeowners, we need to disable this test to unblock CI.

Reply:
> Because zombie processes are a relatively common ocurrence on Windows: any 3rd party process can create one by opening a handle to a process and not closing it when the other process terminates

This makes sense. I think its reasonable to limit the scope of this change to windows machines.